Core/include/Acts/EventData/SpacePointContainer.hpp

```diff
@@ -22,41 +22,21 @@ namespace Acts {
 
 struct SpacePointContainerConfig {
   bool useDetailedDoubleMeasurementInfo = false;
-  bool isInInternalUnits = false;
-
-  SpacePointContainerConfig toInternalUnits() const {
-    if (isInInternalUnits) {
-      throw std::runtime_error(
-          "Repeated conversion to internal units for "
-          "SpacePointContainerConfig");
-    }
-    using namespace Acts::UnitLiterals;
-    SpacePointContainerConfig config = *this;
-    config.isInInternalUnits = true;
-    return config;
-  };
+
+  bool isInInternalUnits = true;
+  //[[deprecated("SpacePointContainerConfig uses internal units")]]
```

Comment:
Can't be an actual `[[deprecated]]`?

Reply:
using `[[deprecated]]` immediately breaks the Athena CI right now. I wanted to do it but after seeing that I thought it might be smoother without it for now. what do you think?